cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/sync/class-upload-sync.php

```diff
@@ -329,27 +337,32 @@ public function add_suffix_maybe( $attachment_id, $suffix = null ) {
 			if ( ! is_wp_error( $cld_asset ) && ! empty( $cld_asset['public_id'] ) ) {
 				$context_guid    = null;
 				$attachment_guid = md5( get_the_guid( $attachment_id ) );
-				$saved_version = $this->media->get_post_meta( $attachment_id, Sync::META_KEYS['_cloudinary_version'], true );
 
 				// Exists, check to see if this asset originally belongs to this ID.
 				if ( ! empty( $cld_asset['context'] ) && ! empty( $cld_asset['context']['custom'] ) && ! empty( $cld_asset['context']['custom']['guid'] ) ) {
 					$context_guid = $cld_asset['context']['custom']['guid'];
 				}
 
 				// Check to see if this asset originally belongs to this ID using the older wp_id.
-				if ( 
-					! empty( $cld_asset['context'] ) && 
-					! empty( $cld_asset['context']['custom'] ) && 
-					! empty( $cld_asset['context']['custom']['wp_id'] ) && 
-					(int) $cld_asset['context']['custom']['wp_id'] === $attachment_id 
+				if (
+					! empty( $cld_asset['context'] ) &&
```

Comment:
@DavidCramer 

Since we are updating this piece of code, this check and the following are pointless, as the `! empty( $cld_asset['context']['custom']['wp_id'] )` takes care of all :)

Reply:
@pereirinha - This is no longer here- it's been removed already.